nav2_map_server/include/nav2_map_server/map_server.hpp

```diff
@@ -138,6 +138,9 @@ class MapServer : public nav2_util::LifecycleNode
 
   // The message to publish on the occupancy grid topic
   nav_msgs::msg::OccupancyGrid msg_;
+
+  // true if msg_ was initialized
+  bool map_available_;
```

Comment:
Just curious: if we will make `msg_` to be a shared pointer `nav_msgs::msg::OccupancyGrid::SharedPtr`, we could initialize it with `nullptr`. And this will be an indicator, whether the map is available or not. Boolean `map_available_` is not needed in this case.